src/window.rs

```diff
@@ -195,7 +231,52 @@ pub fn draw_gui() {
                                 VirtualKeyCode::Right => {
                                     input_x.y = 0.0;
                                 }
-                                _ => {}
+                                k => {
```

Comment:
nit: macro?

Reply:
I'm not sure how a macro would help us here, but we should definitely figure something out to clean it up in the future.